src/subs/FitSubroutines.cpp

```diff
@@ -1650,6 +1651,23 @@ void readColors(const img::FTable &extensionTable,
     }  // end loop over catalogs to read
 }
 
+// Get color information from an input vector of colors matched to the current matches.
+template <class S>
+void readColors(typename S::MCat &matches,
+                const std::vector<int> matchIDs,
+                const std::vector<double> colors) {
+    vector<typename S::Match *> vmatches;
```

Comment:
Yeah, I convert it for easier access. I can't remember at this point if there's a good reason for `matches` to be a list rather than a vector throughout, but it would require pretty extensive work to change it.